CMakeLists.txt

```
@@ -103,9 +110,9 @@ else()
 
   if(TRITON_ENABLE_GPU)
     rapids_cuda_init_architectures(RAPIDS_TRITON_BACKEND)
-    project(RAPIDS_TRITON_BACKEND VERSION 21.12.00 LANGUAGES CXX CUDA)
+    project(RAPIDS_TRITON_BACKEND VERSION 22.02.00 LANGUAGES CXX CUDA)
```

```suggestion
    project(RAPIDS_TRITON_BACKEND VERSION 22.03.00 LANGUAGES CXX CUDA)
```

shouldn't this be 22.03?